src/StreamJsonRpc.Tests/JsonRpcClientInteropTests.cs

```diff
@@ -44,6 +45,82 @@ public JsonRpcClientInteropTests(ITestOutputHelper logger)
     }
 
     [Fact]
+    public async Task NotifyAsync_ParameterObjectSentAsArray()
+    {
+        Task notifyTask = this.clientRpc.NotifyAsync("test", new  { Bar = "test" });
+        JObject request = await this.ReceiveAsync();
+        Assert.Equal<JTokenType>(JTokenType.Array, request["params"].Type);
+        Assert.Equal<string>("test", ((JArray)request["params"])[0]["Bar"].ToString());
+    }
+
+    [Fact]
+    public async Task NotifyAsync_ParameterNullSentAsArray()
```

Comment:
Your test method name suggests you're passing in a null parameter, but you're not passing in any parameters.